test/requests.js

```diff
@@ -382,6 +382,56 @@ var tests = function(web3) {
 
   });
 
+  describe("eth_ecRecover", function() {
```

Comment:
Given that this isn't a part of the RPC standard, I wonder if this wouldn't be a better contribution to [truffle-contract](https://github.com/trufflesuite/truffle-contract) and/or [drizzle](https://github.com/trufflesuite/drizzle)?

Reply:
I'm referring specifically to the logic contained in this function - obviously the other changes would still be necessary for that to work.

Reply:
I think this should be personal_ecRecover and there should be a corresponding personal_sign method, is that right?